### PR TITLE
Start tracker with other agents.

### PR DESCRIPTION
**Pull request by amine3**

### CODE REVIEW

File: src/ostorlab/runtimes/local/runtime.py [hunk 1]
```diff
@@ -204,6 +204,10 @@ def scan(
             console.info("Starting agents")
             self._start_agents(agent_group_definition)
 
+            if self._run_default_agents is True:
```
Comment (mohsinenar) [inline src/ostorlab/runtimes/local/runtime.py:207]: How about starting all the default agents at once? can you try that?

Comment (amine3) [reply src/ostorlab/runtimes/local/runtime.py:207]: This is what we do with this config, we start pre-agents, the agents and post-agents at the same time.

Comment (mohsinenar) [reply src/ostorlab/runtimes/local/runtime.py:207]: @amine3 what I mean is to start pre-agents and post-agents in the same call! 
```python
 if self._run_default_agents is True:
                console.info("Starting pre-agents")
                self._start_pre_agents()
                self._start_post_agents()
```

Comment (3asm) [reply src/ostorlab/runtimes/local/runtime.py:207]: The separation makes is semantic to differentiate between the agents' types. Once the flow is confirmed to work. We may merge API.

**Review by 3asm — APPROVED** [review]

**Pull request merged**